neos/neos-development-collection
# FEATURE: Change wallpaper for login

**What I did**
Replaces the JPG and WebP file of the wallpaper.

**How I did it**
Used imageOptim to minimize the image from @kdambekalns with Gruetzli and convert the image also to a WebP.

**How to verify it**
Check the wonderful new login screen.
Thanks to @kdambekalns for providing such a nice picture from Saxony-Anhalt (Germany) ❤️ 

Resolves: #3522

![Screenshot 2021-12-07 at 00 54 19](https://user-images.githubusercontent.com/1014126/144941468-55cb4a29-f073-4c5c-b9cd-7ce90a8149f5.png)
